src/trio/_core/_tests/test_ki.py

```diff
@@ -165,7 +165,7 @@ def protected_manager() -> Iterator[None]:
 async def test_async_generator_agen_protection() -> None:
     @_core.enable_ki_protection
     @async_generator  # type: ignore[misc] # untyped generator
-    async def agen_protected1() -> None:
+    async def agen_protected1() -> None:  # type: ignore[misc] # untyped generator
```

Comment:
huh, I'm surprised we need this on both L167 and L168

Reply:
Yeah, but it is indeed needed :-/